applications/IgaApplication/tests/test_IgaApplication.py

```diff
@@ -60,6 +65,16 @@ def AssembleTestSuites():
         SinglePatchRefinedSupportLagrangeTest
         ]))
 
+    if has_linear_solvers_application:
+        from KratosMultiphysics import LinearSolversApplication
+        if LinearSolversApplication.HasFEAST():
+            smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([
+                # Weak support Nitsche test
+                SinglePatchRefinedSupportNitscheTest
+                ]))
+        else:
+            print("FEAST not available in LinearSolversApplication")
+
```

Comment:
👍

Reply:
Yeah, we need this as FEAST doesn't work on Windows OS. The next step is to figure out how to run the Nitsche condition on Windows, perhaps try/use another solver. I'll save it for later.